packages/rum/src/domain/record/observers.ts

```diff
@@ -151,23 +152,26 @@ export function initMutationObserver(
   return startMutationObserver(cb, configuration, shadowRootsController, document)
 }
 
-function initMoveObserver(cb: MousemoveCallBack): ListenerHandler {
+export function initMoveObserver(cb: MousemoveCallBack): ListenerHandler {
   const { throttled: updatePosition } = throttle(
     monitor((event: MouseEvent | TouchEvent) => {
       const target = getEventTarget(event)
       if (hasSerializedNode(target)) {
-        const { clientX, clientY } = isTouchEvent(event) ? event.changedTouches[0] : event
+        const { x, y } = computeCoordinates(event)
+        if (!Number.isFinite(x) || !Number.isFinite(y)) {
+          addTelemetryDebug('mouse/touch event without x/y', {
+            isTrusted: event.isTrusted,
+            position: { x, y },
+          })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ What will those telemetry logs used for?

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ If it's to see whether ignoring untrusted events would help, I would suggest emiting a telemetry log only when the event is untrusted. This would reduce the amount of unneeded logs messages.

```suggestion
          if (event.isTrusted) {
            addTelemetryDebug('mouse/touch event without x/y', {
              position: { x, y },
            })
          }
```

Reply:
I disagree, if we have event without x/y that are both trusted and not trusted I would like to know about it

Reply:
> What will those telemetry logs used for?

I want to know if the next major release will fix this issue for and we can remove that code

Reply:
We know that some untrusted events don't have coordinates. I don't think we need to confirm that anymore, since we can easily reproduce it.

Now, what I'm worried about is the amount of telemetry logs that could be sent. If you still think that we should add a log when an untrusted event don't have coordinates, I would suggest to limit the amount of telemetry debug by only logging the first one, something like:

```ts
let wasEventWithoutCoordinatesLogSent = false

function tryComputeCoordinates(event) {
  ...
  if (!wasEventWithoutCoordinatesLogSent && !Number.isFinite(...) && ...) {
    wasEventWithoutCoordinatesLogSent = true
    addTelemetryDebug('mouse/touch event without x/y', {
      isTrusted: event.isTrusted,
      position: { x, y },
    })
  }
  ...
}
```

Reply:
Alright, I see your point, I will only send the telemetry if event is trusted